explorer/__init__.py

```diff
@@ -1,7 +1,7 @@
 __version_info__ = {
-    "major": 3,
-    "minor": 2,
-    "patch": 1,
+    "major": 4,
+    "minor": 0,
+    "patch": 0,
     "releaselevel": "final",
```

Comment:
Might be worth doing a beta release first, just because the changes are so big.

Reply:
Yeah, probably a good idea. I will do that. And I'll get my company to adopt it and test it :)